frontend/fingertips-frontend/playwright/tests/search.spec.ts

```diff
@@ -22,4 +22,29 @@ test('Search via indicator and assert results', async ({
   await resultsPage.checkURLIsCorrect(indicator);
   expect((await axeBuilder.analyze()).violations).toEqual([]);
   await resultsPage.checkSearchResults(indicator);
+  await resultsPage.checkResultsAreDisplayed();
+});
+
+test('Should return to search page upon results page back link click', async ({
+  searchPage,
+  resultsPage,
+  axeBuilder,
+}) => {
+  // Arrange
+  await searchPage.navigateToSearch();
+
+  // Act
+  await searchPage.typeIndicator(indicator);
```

Comment:
can we add an // Assert section to call

  await searchPage.checkURLIsCorrect();

or is this overkill?

Reply:
I didn't add this as it seems the first test already does that check upon initial navigation to the search page, but I'm happy to add it in.